skin/src/main/java/org/researchstack/skin/ui/fragment/SettingsFragment.java

```diff
@@ -313,6 +318,7 @@ public boolean onPreferenceTreeClick(Preference preference)
                                                                 R.string.rss_network_result_consent_withdraw_success,
                                                                 Toast.LENGTH_SHORT).show();
 
+                                                        isInitializedForConsent = false;
```

Comment:
Do we need to worry about setting this to `true` if it fails (i.e. in the `else` or `onError`)?